stopwatch.py

```diff
@@ -36,6 +36,9 @@
 import random as insecure_random
 import time
 
+TraceAnnotation = collections.namedtuple('TraceAnnotation', ['name', 'time'])
+TraceKeyValueAnnotation = collections.namedtuple('TraceKeyValueAnnotation', ['key', 'value'])
```

Comment:
I think that the fact that the library exposes two different ways of providing annotations is pretty bizarre.
I think we can go with this temporarily to be compatible, but I think it makes more sense to have one kind of annotation with a key/value/time

addtag would simply be a key, with a value of '1' and a time of _time_func()
trace_root_event would also simply be a key/value _time_func.
Finally, the more advanced key/value could also log as events.

Thus we can consolidate those three functions in this library into one smart-annotation.
What do you think? I'll make another pull request that attempts that approach.